jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/JmxConfig.java

```diff
@@ -32,6 +32,16 @@ class JmxConfig {
   static final String JMX_REMOTE_PROFILE = PREFIX + "jmx.remote.profile";
   static final String JMX_REALM = PREFIX + "jmx.realm";
 
+  static final List<String> JAVA_SYSTEM_PROPERTIES =
+      Arrays.asList(
+          "otel.resource.attributes",
```

Comment:
About `otel.resource.attributes`: take a look at https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/GroovyMetricEnvironment.java#L63-L85 -- according to that code, the resource property is already being propagated to the OTel SDK autoconfiguration. Does this not work?

Reply:
I had missed this component of the groovy metric environment startup! Thanks for pointing it out, I'll have to remove the otel.resource.attributes from the list that gets relocated into system.properties.